pulp_rpm/tests/functional/api/test_fips_workflow.py

```diff
@@ -1,963 +1,212 @@
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/ansible/2.5/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhgs-server-nfs/3.1/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7.6/x86_64/kickstart",
+        "https://cdn.redhat.com/content/dist/rhel/workstation/7/7.5/x86_64/kickstart",
+        "https://mirrors.kernel.org/fedora-epel/8/Everything/x86_64/",
+        "https://cdn.redhat.com/content/dist/rhel/workstation/7/7Workstation/x86_64/insights/3/os",
+        "https://cdn.redhat.com/content/dist/rhel/workstation/7/7Workstation/x86_64/rh-common/os",
+        "https://cdn.redhat.com/content/dist/rhel/workstation/7/7Workstation/x86_64/os",
+        "https://cdn.redhat.com/content/dist/rhel/workstation/7/7Workstation/x86_64/extras/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rh-gluster-samba/3.1/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhgs-server/3.1/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/6/6.10/x86_64/kickstart",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhgs-nagios/3.1/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhscon-agent/2/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhscon-main/2/os",
+        "http://vault.centos.org/6.10/updates/x86_64/",
+        "https://cdn.redhat.com/content/dist/rhel/server/6/6Server/x86_64/rhs-client/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhs-client/os",
+        "http://mirror.centos.org/centos-7/7/sclo/x86_64/rh/",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/supplementary/os",
+        "https://cdn.redhat.com/content/dist/rhel/workstation/7/7.6/x86_64/kickstart",
+        "http://mirror.centos.org/centos-7/7/updates/x86_64/",
+        "https://cdn.redhat.com/content/dist/rhel/server/6/6.8/x86_64/kickstart",
+        "https://cdn.redhat.com/content/dist/rhel/server/6/6.9/x86_64/kickstart",
+        "https://cdn.redhat.com/content/eus/rhel/server/6/6.6/x86_64/sat-tools/6.2/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhscl/1/os",
+        "http://mirror.centos.org/centos-7/7/os/x86_64/",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/ansible/2.7/os",
+        "https://cdn.redhat.com/content/eus/rhel/server/7/7.3/x86_64/os",
+        "https://cdn.redhat.com/content/eus/rhel/server/7/7.6/x86_64/os",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7.4/x86_64/kickstart",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7.5/x86_64/kickstart",
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhgs-server-bigdata/3.1/os",  # noqa D102
+        "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhgs-server-splunk/3.1/os",  # noqa D102
```

Comment:
Also these `# noqa D102` aren't relevant anymore

Reply:
@gerrod3 ^

Reply:
They are still needed since these lines are 103 characters & and I didn't want to break them into multiple lines.

Reply:
I think D102 is "missing docstring in public function", though, not line-length

Reply:
You are right, it should have been 501.